lib/logging.dart

```diff
@@ -364,3 +366,187 @@ class LogRecord {
   @override
   String toString() => '[${level.name}] $loggerName: $message';
 }
+
+typedef _ServiceExtensionCallback = Future<Map<String, dynamic>> Function(
+    Map<String, String> parameters);
+
+/// A shared manager instance whose recorded messages are logged to the
+/// developer console.
+final LogManager logManager = LogManager()..onRecord.listen((record) {
+  developer.log(
+    record.message,
+    name: record.loggerName,
+    time: record.time,
+    error: record.error,
+    level: record.level.value,
+    stackTrace: record.stackTrace,
+  );
+});
+
+/// Log Managers record messages only written to loggers explicitly enabled by
+/// name.
+///
+/// For example, in the given source, only messages to `loggerA` will be sent to
+/// the `onRecord` stream.
+///
+/// ```
+/// var loggerA = Logger('loggerA');
+/// var loggerA = Logger('loggerB');
+///
+/// var manager = LogManager()..enableLogging('loggerA');
+///
+/// loggerA.info('hello from logger A'); // recorded
+/// loggerB.info('here goes nothing');   // ignored
+/// ```
+///
+/// Log managers can be created but the provided [logManager] should suffice
+/// for most purposes.  The shared manager redirects recorded messages to the
+/// developer log.
+///
+/// **Important note:** managed loggers respect logger `root` level configuration.
+/// For example, in the following, `logger` only reports `severe` messages
+/// as expected.
+///
+/// ```
+/// Logger.root.level = Level.SEVERE;
+///
+/// ...
+///
+/// var logger = Logger('my.logger');
+/// logManager.enable('my.logger');
+/// try {
+///   logger.info('doing something'); // ignored
+///   ...
+/// } catch(e, stackTrace) {
+///   logger.severe('uh-oh!, e, stackTrace); // recorded
+/// }
+/// ```
+///
+/// * To create loggers, see [Logger].
```

Comment:
Nit: considering writing this comment as:

/// See also:
///
/// * [Logger], to create a logger.
/// * [enableLogging], to enable or disable a logger.

to get nice dart doc formatting.

Reply:
Good deal.  Updated.